extensions/amp-story-dev-tools/0.1/amp-story-dev-tools.css

```diff
@@ -18,10 +18,85 @@ amp-story-dev-tools {
   width: 100% !important;
   height: 100% !important;
   box-sizing: border-box !important;
-  background-color: #121212 !important;
+  background-color: black !important;
   font-size: 16px !important;
   font-family: 'Poppins', sans-serif !important;
   color: white !important;
   padding: 30px 10% !important;
   text-align: center !important;
 }
+
+.i-amphtml-story-dev-tools-container {
+  height: 100% !important;
+}
+
+.i-amphtml-story-dev-tools-header {
+  display: flex !important;
+  align-items: center !important;
+  justify-content: space-between !important;
+  color: white !important;
+  height: 40px !important;
+  box-sizing: border-box !important;
+}
+
+.i-amphtml-story-dev-tools-brand {
+  display: inline-flex !important;
```

Comment:
I think this could be just `flex`.

Reply:
Yeah they are both essentially the same... I guess `flex` has less characters so we'd save some bytes 😁